models/Task.java

```diff
@@ -0,0 +1,27 @@
+package com.codesoom.assignment.models;
+
+public class Task {
+    private Long id;
+
+    private String title;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public String toString() {
+        return "Task - title: " + title;
+    }
```

Comment:
이런 override 메소드는 메소드 시그니처 위에 `@Override` 애노테이션을 달아주는 것이 관례입니다.
```suggestion
    @Override
    public String toString() {
        return "Task - title: " + title;
    }
```

한편, 학습하시는 김에 공식 문서의 `toString` 메소드 JavaDoc도 읽어보시면 큰 도움이 될 것 같습니다.

https://docs.oracle.com/en/java/javase/15/docs/api/java.base/java/lang/Object.html

문서가 너무 어렵다면 다음 글도 참고해주세요. 제 블로그에 작성한 문서입니다.
https://johngrib.github.io/wiki/Object-toString/